guides/source/action_cable_overview.md

```diff
@@ -235,10 +235,58 @@ class ChatChannel < ApplicationCable::Channel
   # Called when the consumer has successfully
   # become a subscriber to this channel.
   def subscribed
+    # stream_from "some_channel"
+  end
+
+  # Called once a consumer has cut its cable connection.
+  def unsubscribed
+    # Any cleanup needed when channel is unsubscribed
   end
 end
 ```
 
+#### Actions
+
+Almost any public method declared on a channel is automatically exposed as a
+callable action to the client. These methods can take an optional `data`
+argument containing an optional Hash sent by the client.
+
+The `#subscribed` and `#unsubscribed` methods , and any public method
+defined on `ActionCable::Channel::Base`, can not be called by the client.
+
+Example:
+
+```ruby
+class AppearanceChannel < ApplicationCable::Channel
+  def subscribed
+    @connection_token = generate_connection_token
+  end
+
+  def unsubscribed
+    current_user.disappear @connection_token
+  end
+
+  def appear(data)
+    current_user.appear @connection_token, on: data['appearing_on']
+  end
+
+  def away
+    current_user.away @connection_token
+  end
+
+  private
+    def generate_connection_token
+      SecureRandom.hex(36)
+    end
+end
+```
+
+In this example `#appear` and `#away` are callable by the client, whereas
+`#subscribed` and `#unsubscribed` are not. `#generate_connection_token` is also
+not callable, since it's a private method. You'll see that `#appear` accepts a
+`data` parameter, which it then uses as part of its model call. `#away` does
+not, since it's simply a trigger action.
```

Comment:
I think this example comes from here:
https://edgeapi.rubyonrails.org/classes/ActionCable/Channel/Base.html

Which is fine, but I think if we're going to update the description let's do it in both places.

e.g. (without code formatting, from copy/paste)

> In this example, the subscribed and unsubscribed methods are not callable methods, as they were already declared in [ActionCable::Channel::Base](https://edgeapi.rubyonrails.org/classes/ActionCable/Channel/Base.html), but #appear and #away are. #generate_connection_token is also not callable, since it’s a private method. You’ll see that appear accepts a data parameter, which it then uses as part of its model call. #away does not, since it’s simply a trigger action.
> 
> Also note that in this example, current_user is available because it was marked as an identifying attribute on the connection. All such identifiers will automatically create a delegation method of the same name on the channel instance.